layers/+email/mu4e/packages.el

```diff
@@ -37,7 +37,7 @@
       (setq mu4e-completing-read-function
             (if (configuration-layer/layer-usedp 'spacemacs-ivy)
                 'ivy-completing-read
-              'helm-completing-read))
+              'helm--completing-read-default))
```

Comment:
~~~
This function should be used only as a `completing-read-function'.
~~~

In this case, I'm using it as a `completing-read-function`.

~~~
Don't use it directly, use instead `helm-comp-read' in your programs.
~~~

This is talking about packages and functions using helm directly for completing read.

Or the cop-out answer: `helm--completing-read-default` works and `helm-comp-read` doesn't.